readme.md

```diff
@@ -50,7 +50,7 @@ const run = async () => {
 (async () => {
 	const result = await pRetry(run, {
 		onFailedAttempt: error => {
-			console.log(`Attempt ${error.attemptNumber} failed. There are ${error.attemptsLeft} attempts left.`);
+			console.log(`Attempt ${error.attemptNumber} failed. There are ${error.retriesLeft} attempts left.`);
```

Comment:
If you are replacing that, should would think you also should replace the word `attempts` with `retries` ?

Reply:
@pragmaticivan Yeah, It makes sense.

Thanks for pointing it out. I've updated my PR with that change.